### DISCUSSION TITLE
Repository location for API documentation?

### DISCUSSION THREAD

**Discussion by feenst** [community/community#21548] [New to GitHub]
<p>I found something unclear in the documentation and would like to open a pull request to update, but I can’t find an associated repository.</p>
<p>Where can I find the repository?</p>

**Comment by JulienBreux** [comment, marked as answer]
<p>Hi <a class="mention" href="/feenst">@feenst</a>,</p>
<p>If you speak about Github documentation, you must contact the support <a href="https://github.com/contact" rel="nofollow noopener">here</a>.</p>
<p>I wish you better :wink: Have a good day.</p>
